DataManagementSystem/Client/DataManager.py

```diff
@@ -812,54 +811,74 @@ def __replicate( self, lfn, destSEName, sourceSEName = '', destPath = '', localC
         continue
 
 
-      replicationProtocol = res['Value']
+      replicationProtocols = res['Value']
 
-      if not replicationProtocol:
+      if not replicationProtocols:
         possibleIntermediateSEs.append( candidateSE )
         log.debug( "No protocol suitable for replication found" )
         continue
 
-      log.debug( 'Found common protocols', replicationProtocol )
+      log.debug( 'Found common protocols', replicationProtocols )
 
       # THIS WOULD NOT WORK IF PROTO == file !!
+      # Why did I write that comment ?!
```

Comment:
Yes it relates to what is before, but I am sure there was a reason, and I can't figure out why :-(